### PR TITLE
Update PROJECT_NAME_graduation_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
suggested changes to graduated projects template

### CODE REVIEW

**Comment by camaleon2016** [comment]
What do we consider a "consistent release cadence" to be? I believe this needs to be less subjective. Are we thinking at least once a year? every couple of years?

**Comment by camaleon2016** [comment]
Way may want to have them either point to a Specific SLSA Track in the framework, point to the use of a different framework and justify why. If the desire is to use the SLSA framework then the track used and adherence to the track should be discussed in previous stages. Consider feasibility stage and security requirements.

**Review by camaleon2016 — APPROVED** [review]

File: process/templates/PROJECT_NAME_graduation_stage.md [hunk 1]
```diff
@@ -14,23 +22,41 @@ The project must be able to show adoption by multiple parties, which could be pr
...
+Projects should harden their build systems in accordance with the SLSA Framework
+  * "link to SLSA documentation"
 
 ### Security audit
 When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.
   * "link to results of security audit"
 
+### To retain Graduated status
+A TI is expected to continue operating per the above guidelines, and to provide the TAC with quarterly status updates, and to approach the TAC when seeking approval of substantial changes (such as when accepting or promoting new Projects).
+
```
[the remark this answers is not in the document]
Comment (SecurityCRob) [reply process/templates/PROJECT_NAME_graduation_stage.md:38]: I prefer to keep this addition to ensure TI continues to be active

**Review by steiza — APPROVED** [review]
Thanks!

**Review by bobcallaway — APPROVED** [review]

File: process/templates/PROJECT_NAME_graduation_stage.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 ## Project graduation application
 
+### Project has met all Incubating requirements
+  * "link to Incubating PR"
```
Comment (lehors) [inline process/templates/PROJECT_NAME_graduation_stage.md:4]: Same comment as before: I think there is no need to ask people to dig this up.

**Review by lehors — CHANGES_REQUESTED** [review]

File: process/templates/PROJECT_NAME_graduation_stage.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+
 ### List of project maintainers
-The project must have maintainers with a minimum of three different organizational affiliations.
+The project must have maintainers with a minimum of five different contributors from three different organizational affiliations.
```
Comment (marcelamelara) [inline process/templates/PROJECT_NAME_graduation_stage.md:7]: I assume we want to update the requirements in the [project lifecycle doc](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#graduated-project-entry-requirements-and-considerations) to be this specific as well?

File: process/templates/PROJECT_NAME_graduation_stage.md [hunk 4]
```diff
@@ -14,23 +17,36 @@ The project must be able to show adoption by multiple parties, which could be pr
 
 ### Release cadence
 The project must be able to show a consistent release cadence.
+  * "link to releases documentation"
 
 ### Governance
 Projects must have documented project governance and be able to demonstrate that governance in action.
   * "link to governance documents"
+Have a define roadmap and annual goals for the project
```
Comment (marcelamelara) [inline process/templates/PROJECT_NAME_graduation_stage.md:25]:
```suggestion
Have a defined and documented roadmap and annual goals for the project
```

File: process/templates/PROJECT_NAME_graduation_stage.md (hunk 3 above)
Comment (marcelamelara) [inline process/templates/PROJECT_NAME_graduation_stage.md:7]: I assume we want to update the requirements in the [project lifecycle doc](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#graduated-project-entry-requirements-and-considerations) to be this specific as well?

**Review by marcelamelara — COMMENTED** [review]
Some additional comments.

**Review by sevansdell — APPROVED** [review]

File: process/templates/PROJECT_NAME_graduation_stage.md (hunk 3 above)
Comment (SecurityCRob) [reply process/templates/PROJECT_NAME_graduation_stage.md:7]: yes, please

File: process/templates/PROJECT_NAME_graduation_stage.md (hunk 2 above)
Comment (SecurityCRob) [reply process/templates/PROJECT_NAME_graduation_stage.md]: added verbiage arnaud requested

**Comment by SecurityCRob** [comment]
6 of 9 tac members approved, merging

**Pull request merged**